rllib/BUILD

```diff
@@ -833,6 +843,82 @@ py_test(
 )
 
 
+# --------------------------------------------------------------------
+# Memory leak tests
+#
+# Tag: memory_leak_tests
+# --------------------------------------------------------------------
+
+py_test(
+    name = "test_memory_leak_a3c",
+    tags = ["team:ml", "memory_leak_tests"],
+    main = "tests/run_memory_leak_tests.py",
+    size = "large",
+    srcs = ["tests/run_memory_leak_tests.py"],
+    data = ["tuned_examples/a3c/memory-leak-test-a3c.yaml"],
```

Comment:
another way to organize these may be to group these memory-leak-test-xxx.yaml files into their own dir under rllib/tests/memory_leak_tests/.
that way they are not part of the tuned_example folder, since they are mostly for testing purpose, not demonstration purpose.
just a suggestion.